.zenodo.json

```
@@ -0,0 +1,138 @@
+{
+    "creators": [
+      {
+        "affiliation": "pyOpenSci",
+        "name": "Wasser, Leah",
+        "orcid": "0000-0002-8177-6550"
+      },
+      {
+        "affiliation": "",
+        "name": "Nicholson, David",
+        "orcid": "0000-0002-4261-4719"
+      },
+      {
+        "affiliation": "Hasso Plattner Institute",
+        "name": "Sasso, Ariane",
+        "orcid": "0000-0002-3669-4599"
+      },
+      {
+        "affiliation": "",
+        "name": "Batisse, Alexandre",
+        "orcid": "0000-0001-7912-3422"
+      }
+      {
+        "affiliation": "",
+        "name": "Molinsky, Simon",
+        "orcid": ""
+      },
+    ],
+    "contributors": [
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Marmo, Chiara",
+          "orcid": "0000-0003-2843-6044 "
+        },
+        {
+          "type":"Other",
+          "affiliation": "Quansight",
+          "name": "Pamphile T., Roy",
+          "orcid": "0000-0001-9816-1416"
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Welch, Erik",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Fernandes, Filipe",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Saunders, Jonny",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Döring, Randy",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Cano Rodríguez, Juan Luis",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Schreiner, Henry",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Van der Walt, Stéfan",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Schwartz, Eli",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Gommers, Ralf",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Gedam, Pradyun",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Lef, Ofek",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Tocknell, James",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Ming, Frost",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "van Kemenade, Hugo",
+          "orcid": ""
```

And mine is https://orcid.org/0000-0001-5715-8632:
```suggestion
          "orcid": "0000-0001-5715-8632"
```